tests/by-util/test_uniq.rs

```diff
@@ -74,7 +74,34 @@ fn test_stdin_skip_and_check_2_chars() {
         .run()
         .stdout_is_fixture("skip-3-check-2-chars.expected");
 }
+/*
```

Comment:
why is that commented ?

Reply:
because the +N option doesn't work yet, i'll uncomment them as soon as i get to implementing it

Reply:
Since the implementation is going to be quite different for those cases maybe we could focus this PR on just `-` and leave `+` for a future PR?